nodejs/week3/src/warmUp3/index3.js

```diff
@@ -0,0 +1,41 @@
+const express = require("express");
+const app = express();
+const PORT = 3000;
+
+app.use(express.json());
+app.use(
+  express.urlencoded({
+    extended: true,
+  })
+);
+
+// Calculator using req.body
+
+app.get("/calculator", (req, res) => {
+  const method = req.body.method;
+  const arrayOfNumbers = Object.values(req.query);
+  const merged = [].concat.apply([], arrayOfNumbers);
+  const numbersToCalculate = merged.map(Number);
```

Comment:
you need to validate that all of these are numbers before performing calculations with them later on. `Number` will return `NaN` if you provide a non valid number to it.

If the client sends you bad data you need to send back an error response. Which one would you send and why?